index.d.ts

```diff
@@ -17,12 +17,12 @@ declare module 'fluture' {
   }
 
   export interface Next<T> {
-    done: false
+    done: boolean | false
```

Comment:
Is the issue also fixed if you do `false as false`?

Reply:
`false as false` is syntax error :\ Setting this to just `boolean`